src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/documents/GotenbergComponentTestBase.kt

```diff
@@ -124,11 +124,11 @@ abstract class GotenbergComponentTestBase : ComponentTestBase(useRealGotenbergSe
       listOf(
         Prisoner(
           prisonerNumber = nomsNumber.value,
+          croNumber = "CRO Num/456",
```

Comment:
I'm just wondering why you re-ordered all these croNumber and pncNumber settings?

Reply:
I initially did a refactor on Prisoner to remove middleName. I'm guessing the IntelliJ function then ordered them according to the constructor.